kolibri/plugins/management/assets/src/views/class-edit-page/user-remove-modal.vue

```diff
@@ -72,13 +70,35 @@
         required: true,
       },
     },
+    computed: {
+      formattedDeleteConfirmation() {
+        return this.$tr(
+          'deleteConfirmation',
+          {
+            username: this.bold(this.username),
+            classname: this.bold(this.classname),
+          }
+        );
+      },
+      formattedAccessReassuranceConfirmation() {
+        return this.$tr(
+          'accessReassurance',
+          {
+            sectionTabName: this.bold(this.$tr('usersTab')),
+          }
+        );
+      },
+    },
     methods: {
       userRemove() {
         this.removeClassUser(this.classid, this.userid);
       },
       close() {
         this.displayModal(false);
       },
+      bold(stringToBold) {
+        return `<strong v-html> ${stringToBold} </strong>`;
```

Comment:
It's an interesting strategy - I like how it hides HTML from the translator.

Might not work as a general pattern in other places because some `$tr` inputs need to be numbers or dates, and passing in an HTML-formatted string instead would break that.

I'll let @rtibbles weigh in also but this looks good for now